src/bot.ts

```diff
@@ -2,13 +2,24 @@
  * Deepindigo
  *
  */
-const { MOVE, SWITCH } = require('leftovers-again/src/decisions');
-
+import ai from 'leftovers-again/src/ai';
+const AI = ai.AI;
+import decisions from 'leftovers-again/src/decisions';
+type Decision = decisions.Decision;
+const MOVE = decisions.MOVE;
+const SWITCH = decisions.SWITCH;
+import battle from 'leftovers-again/src/battle';
+type State = battle.State;
+import pokemon from 'leftovers-again/src/model/pokemon';
+type PokemonData = pokemon.PokemonData;
 /**
  * Your code is pre-built with a very simple bot that chooses a team, then
  * picks randomly from valid moves on its turn.
  */
-class Deepindigo {
+export default class DeepIndigo extends AI {
```

Comment:
i don't gaf how we do it either way, but i will just say that the `AI` class has a `team` function that isn't being enforced here, so somewhere type safety is being lost

Reply:
Nah dude I don't think so, because we're extending a class, not implementing an interface

Reply:
okie